packages/governance/src/types.js

```diff
@@ -598,8 +625,8 @@
 
 /**
  * @callback SetupGovernance
+ * @param {ZoeService} zoe
```

Comment:
I'd be very surprised if this required an actual local `ZoeService`.

```suggestion
 * @param {ERef<ZoeService>} zoe
```